src/providers/fault-count/__mocks__/cat-K-test-data-state-object.ts

```diff
@@ -0,0 +1,89 @@
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/K/partial';
+import { CatKUniqueTypes } from '@dvsa/mes-test-schema/categories/K';
+
+export const catKTestDataStateObject: CatKUniqueTypes.TestData = {
+  drivingFaults: {
+    controlsGears: 1,
+    pedestrianCrossings: 2,
+    ancillaryControls: 1,
+  },
+  seriousFaults: {},
+  dangerousFaults: {},
+  testRequirements: {
+    normalStart1: true,
+    normalStart2: true,
+    angledStart: true,
+    uphillStartDesignatedStart: true,
+  },
+  ETA: {
+    physical: false,
+    verbal: false,
+  },
+  eco: {
+    adviceGivenControl: false,
+    adviceGivenPlanning: false,
+  },
```

Comment:
Not sure if it makes any difference but is controlledStop missing?

Reply:
Great spot. Added controlledStop just as an empty object so it doesn't affect tests.